nucypher/cli/commands/deploy.py

```diff
@@ -604,3 +604,26 @@ def transfer_ownership(general_config, actor_options, target_address, gas):
     else:
         receipts = ADMINISTRATOR.relinquish_ownership(new_owner=target_address, transaction_gas_limit=gas)
         emitter.ipc(receipts, request_id=0, duration=0)  # TODO: #1216
+
+
+@deploy.command("set-range")
+@group_general_config
+@group_actor_options
+@click.option('--minimum', help="Minimum value for range (in wei)", type=WEI)
+@click.option('--default', help="Default value for range (in wei)", type=WEI)
+@click.option('--maximum', help="Maximum value for range (in wei)", type=WEI)
+def set_range(general_config, actor_options, minimum, default, maximum):
+    """
+    Set minimum reward range in policy manager contract.
```

Comment:
I questioned whether it was the range for reward rate, or just the range for the minimum reward rate - just to confirm that it is the latter.

To be more explicit:
```suggestion
    Set the allowed range for the minimum reward rate in the policy manager contract.
```

Reply:
✔️